crates/relayer/relayer-components/src/birelay/impls/auto_birelay.rs

```diff
@@ -171,6 +174,7 @@ where
                 relay: relay_b_to_a.clone(),
                 start_height: start_height_b,
                 end_height: end_height_b,
+                refresh_rate: refresh_rate_b_to_a,
```

Comment:
I am curious why `refresh_rate` is added only for `Source` and not `Destination` tasks ?